srcpkgs/gdm/template

```diff
@@ -56,6 +56,8 @@ post_install() {
 			vsed -i "s/pam_systemd\.so/pam_elogind.so/" "$f"
 		fi
 	done
+	vmkdir etc/profile.d
+	vinstall ${FILESDIR}/zz-gdm.sh 644 etc/profile.d
```

Comment:
Maybe call this `zz-gdm-lang.sh` to make it a bit more obvious from the name what it's doing?